recipes/imguizmo/all/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.4)
+project(imguizmo CXX)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if (WIN32 AND MSVC AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+endif(WIN32 AND MSVC AND BUILD_SHARED_LIBS)
```

- This property is unused on other platforms then Windows.
- This property is only used when building shared libraries
- This property is also useful for mingw.
```suggestion
set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```